src/actions/install-update.js

```diff
@@ -43,7 +44,9 @@ module.exports = function hydrator (params, callback) {
         if (isJs) dir = join(cwd, 'node_modules')
         if (isPy) dir = join(cwd, 'vendor')
         if (isRb) dir = join(cwd, 'vendor', 'bundle')
-        rm(dir, callback)
+        if (isDeno) callback()
```

Comment:
Why would we skip Deno? We definitely need a fresh function when hydrating, we don't know when the last time dependencies were pulled or what state they're in, this prevents a lot of transient dependency issues (and enables Sandbox symlinking).

Reply:
Maybe I've misunderstood - I thought the rm of those dirs was because for the other runtimes the shared & view folders are symlinked into the dependency folder? So like in node you can `let sharedFunc = '@architect/shared/function.js`

I was thinking because in Deno `vendor` is just a standard dir it wouldn't need cleaning up?